src/views/virtualmachines/details/tabs/overview/components/VirtualMachinesOverviewTabNetworkInterfaces/utils/types.ts

```diff
@@ -2,6 +2,9 @@ import { V1Interface, V1Network } from '@kubevirt-ui/kubevirt-api/kubevirt';
 
 export type InterfacesData = {
   iface: V1Interface;
-  ipAddresses: string[];
+  ipAddresses: {
+    interfaceName: string;
+    ip: string;
+  }[];
```

Comment:
I see this type in a few places in this PR, maybe extract it to another type?

Reply:
sure, done